mwzeval/metrics.py

```diff
@@ -293,11 +293,11 @@ def get_dst(input_data, reference_states, include_loocv_metrics=False, fuzzy_rat
     """
```

Comment:
let us clearly document the format for the prediction files here to make it easier for users to use the code. I imagine this is in the README of the repo, but never hurts to have it in the context where unfamiliar users might need to quickly refer to it.